code/modules/mob/living/living.dm

```diff
@@ -630,7 +630,7 @@ default behaviour is:
 		var/speed = movement_delay()
 		resting = TRUE
 		var/dir = C.true_dir
-		if(ishuman(src) && (dir))// If true_dir = 0(src isn't moving), doesn't proc
+		if(ishuman(src) && !src.weakened && (dir))// If true_dir = 0(src isn't moving), doesn't proc
 			var/mob/living/carbon/human/H = src
```

Comment:
What if mob isn't human? More runtimes to runtime god.

Reply:
ishuman ensures the mob is human